### ISSUE TITLE
unreal-doc does not support inline functions

### ISSUE THREAD

**Issue by Packetdancer** [PsichiX/unreal-doc#8]
Attempting to generate documentation for a plugin which contains single-line inline functions in a header file will error out:

```
Error:
  --> 33:26
   |
33 |    FName GetName() const { return VariableName; }
   |                            ^---
   |
   = expected snippet
```

**Comment by NuLL3rr0r** [comment]
True, this is an issue when we try unreal-doc on our plugin's source code. We also have a lot of inline methods.

**Comment by davidchagalidze** [comment]
> True, this is an issue when we try unreal-doc on our plugin's source code. We also have a lot of inline methods.


@NuLL3rr0r , I noticed you mentioned that using unreal-doc on your plugin’s source code, especially with the inline methods, was an issue. Were you able to find a fix for that?

**Comment by NuLL3rr0r** [comment]
Nope, we decided to document everything manually. But, we might fork the tool for future use as we have lots of API.
